seed/lib/mappings/data/pm-mapping.json

```diff
@@ -441,17 +666,26 @@
     "schema": ""
   },
   {
-    "display_name": "State Province",
```

Comment:
Not sure we want to delete this. Though it is interesting that `state_province` doesn't exist in the PropertyState model... @nllong are we intentionally mapping to extra data fields for this?